yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/JsonResourceFetcher.java

```diff
@@ -1,66 +1,97 @@
 package com.yoti.api.client.spi.remote.call;
 
-import static java.net.HttpURLConnection.HTTP_OK;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.HttpURLConnection;
 import java.util.Map;
 import java.util.Scanner;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
+import static java.net.HttpURLConnection.HTTP_OK;
 
 final class JsonResourceFetcher implements ResourceFetcher {
+
     private static final Logger LOG = LoggerFactory.getLogger(JsonResourceFetcher.class);
-    private static final String HTTP_GET_METHOD = "GET";
-    private static final ObjectMapper JSON_MAPPER = new ObjectMapper();
+    private static final String HTTP_GET = "GET";
+    private static final String HTTP_POST = "POST";
     private static final String DEFAULT_CHARSET = "UTF-8";
 
+    private final ObjectMapper objectMapper;
+
+    public static JsonResourceFetcher createInstance() {
+        return new JsonResourceFetcher(new ObjectMapper());
+    }
+
+    JsonResourceFetcher(ObjectMapper json_mapper) {
```

Comment:
Do you mind if you can change it as camel case and not snake case? thanks.

Reply:
This is already fixed in the next commit